sensor_msgs/PointCloud[] clusters
---
string[] labels

